react-big-calendar/index.d.ts

```diff
@@ -26,10 +26,10 @@ declare module 'react-big-calendar' {
         step?: number;
         rtl?: boolean;
         eventPropGetter?: (event: Object, start: stringOrDate, end: stringOrDate, isSelected: boolean) => void;
-        titleAccessor?: string;
-        allDayAccessor?: boolean;
-        startAccessor?: stringOrDate;
-        endAccessor?: stringOrDate;
+        titleAccessor?: string | ((row: Object) => string);
+        allDayAccessor?: string | ((row: Object) => boolean);
+        startAccessor?: string | ((row: Object) => Date);
+        endAccessor?: string | ((row: Object) => Date);
```

Comment:
This is the same issue of the **string|()=>bool** type guard issue.  "myDateFieldName" is not a date, it's a string.

I did try to remove the extra parens before the original commit, I don't like extra parens either.  It won't build without them, TS gets confused about the end of the statement.